io.github.dweymouth.supersonic.yml

```
@@ -13,6 +13,7 @@ sdk-extensions:
   - org.freedesktop.Sdk.Extension.golang
 command: supersonic
 rename-desktop-file: Supersonic.desktop
+rename-icon: Supersonic.png
```

```suggestion
rename-icon: Supersonic
``` 

no extension. That will also fix the .desktop

but the icon is still in the wrong path.

---

> but the icon is still in the wrong path.

what would be the proper path? here it's in `/appshare/pixmaps/Supersonic.png`, what's the correct one?

---

it has to follow the icon theme spec https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#directory_layout

minimum size is 128x128.

---

ideally it would also be fixed upstream.

---

i tried to move the icon to the right location in [0ea2517](https://github.com/flathub/flathub/pull/4073/commits/0ea2517ce7d1fe6ae6734afa2fca96511548032c) but now the build fails with

```
Error: icon Supersonic not found below /home/anarcat/src/flathub/.flatpak-builder/rofiles/rofiles-qg3xRa/files/share/icons
```

not sure what i am doing wrong now...

---

fixed and reported upstream as https://github.com/dweymouth/supersonic/issues/132